src/crypto/crypto.cpp

```diff
@@ -242,6 +249,126 @@ namespace crypto {
     return sc_isnonzero(&c) == 0;
   }
 
+  void crypto_ops::generate_tx_proof(const hash &prefix_hash, const public_key &R, const public_key &A, const public_key &D, const secret_key &r, signature &sig) {
+    boost::lock_guard<boost::mutex> lock(random_lock);
```

Comment:
If you want to change those, either a separate PR, or the same PR in a different commit.